### PR TITLE
15380 OWO - Multiple Language Support

### PR DESCRIPTION
**Pull request by kyletran-accesso**
Adding 4 new files - French, German, Italian, Portugese US 
Spanish US exists, but the locale ID is not correct. I changed it to es-US

### CODE REVIEW

**Comment by kyletran-accesso** [comment]
Do not merge this yet because I still have a SQL to push up

File: angular-locale_es-us.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ $provide.value("$locale", {
     ]
   },
   "id": "es-us",
-  "localeID": "es_US",
+  "localeID": "es-US",
```
Comment (janamaysaccesso) [inline angular-locale_es-us.js]: Why was this changed to - from _? and why are the rest of these files - rather than _? I don't believe we use this property but we should keep it consistent with the other files
